api/src/main/java/org/openmrs/module/fhir/api/diagnosticreport/handler/LaboratoryHandler.java

```diff
@@ -27,20 +27,47 @@ public LaboratoryHandler() {
 		super();
 	}
 
-    private static final String ServiceCategory = "LAB";
+	private static final String ServiceCategory = "LAB";
 
-    public String getServiceCategory() {
-        return ServiceCategory;
-    }
+	public String getServiceCategory() {
+		return ServiceCategory;
+	}
+
+	@Override
+	public DiagnosticReport getFHIRDiagnosticReport(Encounter omrsiagnosticReport) {
```

Comment:
@surangak What if we have to search by `Name`, `Subject` etc ( [search parameters](http://hl7-fhir.github.io/diagnosticreport.html#4.21.6) ) ?
I think we also have to chagne method name into `getFHIRDiagnosticReportById(String id)`, is it ?